pkg/tarmak/interfaces/interfaces.go

```diff
@@ -271,3 +273,10 @@ type Initialize interface {
 	CurrentProvider() Provider
 	CurrentEnvironment() Environment
 }
+
+type Context interface {
```

Comment:
Not too happy with calling that Context. This name is quite overloaded (Context from Kubeconfig file, golang's `import "context"`, ...

Maybe we can call it `CancelationContext`?

Reply:
done